.github/workflows/test.yaml

```diff
@@ -38,6 +38,12 @@ jobs:
           - package: sqlconnect/internal
             exclude: sqlconnect-go/sqlconnect/internal/(bigquery|databricks|mysql|postgres|redshift|snowflake|trino)
     steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_REDSHIFT_GINKGO_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_REDSHIFT_GINKGO_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
```

Comment:
do we really need this?

Reply:
Yes, it's a requried one. https://github.com/aws-actions/configure-aws-credentials?tab=readme-ov-file#options

Reply:
Why do we need the action in the first place though? Can't we simply go with config-only options?

Reply:
You mean setting AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY env directly?

Reply:
No, I mean setting `config.AccessKeyID` & `config.SecretAccessKey` instead.

Reply:
this comment is relevant
https://github.com/rudderlabs/sqlconnect-go/pull/132/files#r1686340637

Reply:
We want to test `config.RoleARN`. aws-actions/configure-aws-credentials@v4 is just to assume the role.

Reply:
Looks like `aws-actions/configure-aws-credentials@v4` is just to set env variables for AWS_REGION, AWS_ACCESS_KEY_ID & AWS_SECRET_ACCESS_KEY. We are using sts for assuming the role, right?
So what I am saying is that we don't need the action